benchmarks/bench_kernel_pca_solvers_comparison1.py

```diff
@@ -0,0 +1,153 @@
+"""
+=========================================
+Kernel PCA Solvers comparison benchmark 1
```

Comment:
maybe call this time vs n_samples, and the other one time vs n_components?
(also update filenames if you do)

Reply:
great suggestion